twopaneview/TwoPaneView.js

```diff
@@ -0,0 +1,215 @@
+import React, {Fragment, Component} from 'react';
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  ViewProps,
+  Text,
+  StatusBar,
+  Dimensions,
+} from 'react-native';
+import { DualScreenInfo } from 'react-native-dualscreen'
+
+const TPV_Orientation = {
+  Horizontal: 'horizontal',
+  Vertical: 'vertical',
+};
+
+const TPV_PanePriority = {
+  Pane1: 'pane1',
+  Pane2: 'pane2',
+};
+
+const PaneMode = {
+  Auto: 'Auto',
+  Single: 'Single',
+  Dual: 'Dual',
+};
+
+export default class TwoPaneView extends Component {
+  state = {
+    dims: Dimensions.get('window'),
+    spanning: DualScreenInfo.isSpanning, 
+    panePriority: this.props.panePriority,
+    panePriorityVerticalSpanning: this.props.panePriorityVerticalSpanning,
```

Comment:
>panePriorityVerticalSpanning: this.props.panePriorityVerticalSpanning, [](start = 4, length = 70)

Let's remove this property.  It was only put into the original prototype for demo purposes.

Reply:
True, it is not really used as of now.

Reply:
Temporarily removed. May be added back in the future.